Removed `sidecars` and added new top root elements `addtionalContainers` and `additionalInitContainers` (#251)
hivemq/helm-charts: charts/hivemq-platform/templates/NOTES.txt

-----
HiveMQ Platform release "{{.Release.Name}}" installed in namespace "{{.Release.Namespace}}"

Documentation can be found here: https://docs.hivemq.com/hivemq-platform-operator/index.html
{{- if .Values.config.overrideInitContainers }}

*** Warning ***

> `config.overrideInitContainers` value is deprecated and it might be removed in future releases.

Reviewer: ```suggestion
`config.overrideInitContainers` value is deprecated and will be removed in future releases.
```
